eclair-core/src/main/scala/fr/acinq/eclair/router/Router.scala

```diff
@@ -519,7 +520,7 @@ object Router {
   // @formatter:on
 
   // @formatter:off
-  case class SendChannelQuery(chainHash: ByteVector32, remoteNodeId: PublicKey, to: ActorRef, flags_opt: Option[QueryChannelRangeTlv]) extends RemoteTypes
+  case class SendChannelQuery(chainHash: ByteVector32, remoteNodeId: PublicKey, to: ActorRef, replacePrevious: Boolean, flags_opt: Option[QueryChannelRangeTlv]) extends RemoteTypes
```

Comment:
We don't seem to be using `replacePrevious=true` anywhere except in tests?

Reply:
We do, in `PeerConnection` when receiving their `init` message (that's the only case where we replace the previous sync)